From 9a2ef3bfa5781e9268c508cbf393582bfa459404 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Mon, 09 May 2022 18:46:03 +0300
Subject: [PATCH] [SW-0] rom: remove local key reg reference

Public key registers key3 and key4 were initially defined locally as
they were not available in global specs. Later the global specs was
updated to include these registers.

This patch removes the local macro defines.

Change-Id: I1e5e6fe0eb3c252c6a771d4a06891d1a00f97936
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index fde240e..f7f000a 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -615,9 +615,6 @@
 		security_disable_jtag_if();
 }
 
-/* TODO: get newly added key addresses from global specs once it is updated */
-#define mmD0_PSOC_SECURITY_PUBLIC_KEY_3_0		0xC494190
-#define mmD0_PSOC_SECURITY_PUBLIC_KEY_4_0		0xC4941C0
 static void security_store_keys_in_psoc(uint8_t max_keys, uint32_t *hash)
 {
 	uint8_t key_idx, dword;
